packages/nc-gui/composables/useViewData.ts

```diff
@@ -35,6 +35,7 @@ export function useViewData(
   meta: Ref<TableType | undefined> | ComputedRef<TableType | undefined>,
   viewMeta: Ref<ViewType | undefined> | ComputedRef<(ViewType & { id: string }) | undefined>,
   where?: ComputedRef<string | undefined>,
+  xWhere?:  ComputedRef<string | undefined>,
```

Comment:
Here `xWhere` is useless. you don't need to modify this file at all. In fact, you are passing `xWhere` in `useViewData(meta, view, xWhere)` as the third parameter, which is `where` here.